app/controllers/standup_meetings_controller.rb

```diff
@@ -1,7 +1,23 @@
 class StandupMeetingsController < ApplicationController
+  def index
+    @meeting_date = params[:date].nil? ? Date.current : Date.parse(params[:date])
+    @standup_meetings = StandupMeeting.includes(:user, :standup_meeting_group)
+                                      .where(
+                                        meeting_date: @meeting_date,
+                                        standup_meeting_group_id: params[:standup_meeting_group_id]
+                                      )
+    @date_options = StandupMeeting.meeting_dates(params[:standup_meeting_group_id])
```

Comment:
Related to the above, I don't think we even need this if this was only used in the redirect due to there being no meetings on that day.

Reply:
This is used later for moving between dates in https://github.com/agency-of-learning/PairApp/pull/146 as well. I introduced it here because it is also useful for the redirect guard.

Reply:
I don't think we need it there either. The logic should just be +1 or -1 to the current date. Let weekend pages show with no standup meetings.